app/src/test/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModelTest.kt

```diff
@@ -80,5 +81,31 @@ class ProfileViewModelTest : ShouldSpec(
 
 			tested.profile.value shouldBe Loading
 		}
+
+		should("load profile WHEN updateName called") {
```

Comment:
You should consider more dynamic way of testing the same actions for update/fetch profile :) maybe extracting `action`, such as tested.updateName or tested.fetchProfile  will do the job, especially when fetch profile will be also used for retry

Reply:
Done